src/slick.dataview.ts

```diff
@@ -200,6 +200,11 @@ export class SlickDataView<TData extends SlickDataItem = any> implements CustomD
     this.refreshHints = hints;
   }
 
+  /** get extra filter arguments of the filter method */
+  getFilterArgs(): any {
```

Comment:
does TypeScript complain if you remove the return type `any`? I think it's not needed and I wonder if it would be better to use `unknown` on the function below as in `setFilterArgs(args: unknown)`. The reason I'm asking is that if you remove the return type `any` and the SETTER has the `unknown` then it might infer the correct return type of the GETTER. Can you try that out?